diesel_cli/src/main.rs

```diff
@@ -167,7 +167,7 @@ fn run_database_command(matches: &ArgMatches) -> Result<(), crate::errors::Error
     {
         ("setup", args) => {
             let migrations_dir = self::migrations::migrations_dir(matches)?;
-            database::setup_database(args, &migrations_dir)?;
+            database::setup_database(args, &migrations_dir, false)?;
```

Comment:
This changes the current default behavior. We should also use the `--no-default-migration` flag here instead.

Reply:
Done.